index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Kanu Mike C</title>
+  </head>
+  <body>
+    <div class="hero">
+      <header>
+        <div class="logo">My Logo</div>
+        <nav>
+          <a href="#" class="Ham-menu">
+            <img src="./images/Union.png" alt="Ham buger menu" />
+          </a>
+          <div class="desktop-menu">
+            <ul>
+              <li></li>
+              <li></li>
+              <li></li>
+              <li></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <div class="banner">
+        <h1>I'm Mike Glad to see you!</h1>
+        <p class="banner-details">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+
+        <div class="connect">
+          <h2>LET's CONNECT</h2>
+          <ul class="social-icon">
+            <li>
+              <a href="#"> <img src="./images/twitter.svg" alt="" /> </a>
+            </li>
+            <li><img src="./images/Linkedinicon.svg" alt="" /></li>
+            <li><img src="./images/m.svg" alt="" /></li>
+            <li><img src="./images/github.svg" alt="" /></li>
+            <li><img src="./images/hand.svg" alt="" /></li>
+          </ul>
```

Comment:
- [ ] It would be better if you use `<a>` tags instead of `<img>` for each icon inside `<li>`s, so one can click to reach your social media profiles